arviz/plots/hdiplot.py

```diff
@@ -131,6 +133,11 @@ def plot_hpd(
     backend = backend.lower()
 
     # TODO: Add backend kwargs
-    plot = get_plotting_function("plot_hpd", "hpdplot", backend)
-    ax = plot(**hpdplot_kwargs)
+    plot = get_plotting_function("plot_hdi", "hdiplot", backend)
+    ax = plot(**hdiplot_kwargs)
     return ax
+
+
+def plot_hpd(*args, **kwargs):
+    warnings.warn("plot_hdi has been deprecated, please use plot_hdi", DeprecationWarning)
```

Comment:
Pydocstyle complaining about this one method. Maybe add disable or deprecation comment to get a pass